reference/UiTPAS.v2.json

```diff
@@ -1064,7 +1064,7 @@
             }
           },
           "400": {
-            "description": "Bad Request. Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/organizer-not-found\n* https://api.publiq.be/probs/uitpas/invalid-uitpas-number\n* https://api.publiq.be/probs/uitpas/invalid-insz-number\n* https://api.publiq.be/probs/uitpas/school-not-found\n\nThe detail property might include more information for the client developer.\n",
+            "description": "Bad Request. Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/organizer-not-found\n* https://api.publiq.be/probs/uitpas/invalid-uitpas-number\n* https://api.publiq.be/probs/uitpas/invalid-insz-number\n* https://api.publiq.be/probs/uitpas/school-not-found\n* https://api.publiq.be/probs/uitpas/insz-in-token-required (only in specific cases)\n* https://api.publiq.be/probs/uitpas/insz-required (only in specific cases)\n* https://api.publiq.be/probs/uitpas/insz-token-mismatch (only in specific cases)\n\nThe detail property might include more information for the client developer.\n",
```

Comment:
> https://api.publiq.be/probs/uitpas/insz-required (only in specific cases)

This to me again indicates that we are trying to combine too much use cases in one endpoint

Reply:
> https://api.publiq.be/probs/uitpas/insz-token-mismatch (only in specific cases)

By using a separate endpoint for registering yourself, I think we can avoid this error (because then you can get the `inszNumber` from the token?)